### PR TITLE
JAMES-3440 Configuration option to enable EmailQuery view 

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Review by Arsnael — COMMENTED** [review]
Read it

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
Is that still a draft?

**Comment by chibenwa** [comment]
test this please

File: dockerfiles/run/guice/cassandra/destination/conf/jmap.properties [hunk 1]
```diff
@@ -12,3 +12,7 @@ tls.secret=james72laBalle
 # which should be a PEM format file.
 #
 jwt.publickeypem.url=file://conf/jwt_publickey
+
+# Should simple Email/query be resolved against a Cassandra projection, or should we resolve them against ElasticSearch?
+# This enables a higher resilience, but the projection needs to be correctly populated.
```
Comment (rouazana) [inline dockerfiles/run/guice/cassandra/destination/conf/jmap.properties:17]: maybe should explain that it is false by default in the configuration file?

File: src/site/xdoc/server/config-jmap.xml [hunk 2]
```diff
@@ -64,6 +64,11 @@
                     <dt><strong>upload.max.size</strong></dt>
                     <dd>Optional. Configuration max size Upload in new JMAP-RFC-8621.</dd>
                     <dd>Default value: 30M. Supported units are B (bytes) K (KB) M (MB) G (GB).</dd>
+
+                    <dt><strong>view.email.query.enablede</strong></dt>
```
Comment (LanKhuat) [inline src/site/xdoc/server/config-jmap.xml:68]: enabled

**Review by LanKhuat — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**